### ISSUE TITLE
HTML Basics

### ISSUE THREAD

**Issue by LejoLevan** [LejoLevan/Portolfio#1]
This task will utilize a dedicated branch/issue as a blog to document my progress in grasping the fundamentals of HTML. Commits in this branch will consist of both updates to the main website and demonstrations of tests and random applications of my newfound HTML knowledge.

**Comment by LejoLevan** [comment]
A new commit (https://github.com/LejoLevan/Portolfio/commit/08d5f2d1dcd27153166f90785796b866d02c7d03) has been added to the repository, demonstrating the application of HTML Basics in the portfolio. For further details, please refer to the commit itself.

**Comment by LejoLevan** [comment]
Upon implementing the changes from the mentioned commit, this issue will be marked as resolved once I share a summary of my learning. However, it can be reopened in the future if there's a need to revisit or expand upon the fundamental knowledge gained.

**Comment by LejoLevan** [comment]
Summary: After learning the fundamentals of HTML, including paragraphs, headings, bold and italic text formatting, comments, links, images, and file structure, I have gained the ability to create well-structured and visually appealing web pages.

1. Paragraphs: I now understand how to create paragraphs using the < p > tag, allowing me to organize and format textual content in a clear and readable manner.

2. Headings: By using heading tags (< h1 > to < h6 >), I can create hierarchical structures for my content, making it easier for users and search engines to understand the page's layout and importance of different sections.

3. Bold and Italic Text: I have learned to emphasize specific text by using the < strong > tag for bold and < em > tag for italic formatting, which adds visual impact to important words or phrases.

4. Comments: With HTML comments (< !-- ... -- >), I can leave notes within the code that are not visible on the webpage but are useful for documenting or temporarily disabling certain sections.

5. Links: Understanding anchor tags (< a >) and the href attribute enables me to create hyperlinks to other web pages, external resources, or different sections within the same page.

6. Images: I have gained the ability to add visual elements to my web pages by using the < img > tag and specifying the image source (URL or local file) via the src attribute.

7. File Structure: I now grasp the importance of organizing web page files in a logical and coherent manner, including using folders for images, stylesheets, and scripts to improve maintainability and structure.

By mastering these foundational concepts of HTML, I am well-prepared to create dynamic and engaging web pages that adhere to best practices, ensuring an excellent user experience and efficient collaboration with other developers.

**Comment by LejoLevan** [comment]
Summary: After learning the fundamentals of HTML, including paragraphs, headings, bold and italic text formatting, comments, links, images, and file structure, I have gained the ability to create well-structured and visually appealing web pages.

1. Paragraphs: I now understand how to create paragraphs using the < p > tag, allowing me to organize and format textual content in a clear and readable manner.

2. Headings: By using heading tags (< h1 > to < h6 >), I can create hierarchical structures for my content, making it easier for users and search engines to understand the page's layout and importance of different sections.

3. Bold and Italic Text: I have learned to emphasize specific text by using the < strong > tag for bold and < em > tag for italic formatting, which adds visual impact to important words or phrases.

4. Comments: With HTML comments (< !-- ... -- >), I can leave notes within the code that are not visible on the webpage but are useful for documenting or temporarily disabling certain sections.

5. Links: Understanding anchor tags (< a >) and the href attribute enables me to create hyperlinks to other web pages, external resources, or different sections within the same page.

6. Images: I have gained the ability to add visual elements to my web pages by using the < img > tag and specifying the image source (URL or local file) via the src attribute.

7. File Structure: I now grasp the importance of organizing web page files in a logical and coherent manner, including using folders for images, stylesheets, and scripts to improve maintainability and structure.

By mastering these foundational concepts of HTML, I am well-prepared to create dynamic and engaging web pages that adhere to best practices, ensuring an excellent user experience and efficient collaboration with other developers.
